File: src/gestorAplicacion/utileria/Platillo.java
//Clase platillo 
//Autores: Michael Moreno Valoyes
//para tener informacion especifica de los platillos presentes en el menu y poder agregarlos a las ordenes,
//parte importante de la funcionalidad de facturacion.

package gestorAplicacion.utileria;
import java.io.Serializable;

// La clase Platillo contiene los datos del platillo como el tiempo de preparación, su nombre, tipo, ingredientes, precio, identificador, frecuencia que es la cantidad de veces que se ha pedido
public class Platillo implements Serializable{
	
	private static final long serialVersionUID = 9L;
	private int tiempo; //Tiempo estimado que tarda el platillo en estar listo.
	private String nombre;
	private String tipo; //Especifica si el platillo es desayuno, almuerzo o cena
	private String ingredientes;
	private int precio;
	private String identificador; //id númerico de 3 cifras
	private int frecuencia; //cantidad de veces que el platillo fue pedido
	
	
	//Constructor
	public Platillo(int tiempo, String nombre, String tipo, String ingredientes, int precio, String identificador) {
		this.tiempo = tiempo;
		this.nombre = nombre;
		this.tipo = tipo;
		this.ingredientes = ingredientes;
		this.precio = precio;
		this.setIdentificador(identificador);
	}

	//metodos get y set del atributo tiempo
	public int getTiempo() {
		return tiempo;
	}
	
	public void setTiempo(int tiempo) {
		this.tiempo = tiempo;
	}
	
	//metodos get y set del atributo nombre
	public String getNombre() {
		return nombre;
	}
	
	public void setNombre(String nombre) {
		this.nombre = nombre;
	}
	
	//metodos get y set del atributo tipo
	public String getTipo() {
		return tipo;
	}
	
	public void setTipo(String tipo) {
		this.tipo = tipo;
	}
	
	//metodos get y set del atributo ingredientes
	public String getIngredientes() {
		return ingredientes;
	}
	
	public void setIngredientes(String ingredientes) {
		this.ingredientes = ingredientes;
	}
	
	//metodos get y set del atributo precio
	public int getPrecio() {
		return precio;
	}

	public void setPrecio(int precio) {
		this.precio = precio;
	}
	
	//metodos get y set del atributo identificador
	public String getIdentificador() {
		return identificador;
	}

	public void setIdentificador(String identificador) {
		this.identificador = identificador;
	}
	
	//metodos get y set del atributo frecuencia
	public int getFrecuencia() {
		return frecuencia;
	}

	public void setFrecuencia(int frecuencia) {
		this.frecuencia = frecuencia;
	}
}

File: practica2/src/UI/gestorAplicacion/utileria/platillo.py
# Clase platillo
# Autor: Michael Moreno Valoyes
# Para tener la información especifica de los platillos presentes en el menú y poder agregarlo a las ordenes.
# Parte importante de la funcionalidad de facturación.

class Platillo:
    # Para crear el platillo
    def __init__(self, tiempo, nombre, tipo, ingredientes, precio, identificador, frecuencia):
        self.tiempo = tiempo
        self.nombre = nombre
        self.tipo = tipo
        self.ingredientes = ingredientes
        self.precio = precio
        self.identificador = identificador
        self.frecuencia = frecuencia
    # Métodos set y get:

    def getTiempo(self):
        return self.tiempo

    def setTiempo(self, tiempo):
        self.tiempo = tiempo

    def getNombre(self):
        return self.nombre

    def setNombre(self, nombre):
        self.nombre = nombre

    def getTipo(self):
        return self.tipo

    def setTipo(self, tipo):
        self.tipo = tipo

    def getIngredientes(self):
        return self.ingredientes

    def setIngredientes(self, ingredientes):
        self.ingredientes = ingredientes

    def getPrecio(self):
        return self.precio

    def setPrecio(self, precio):
        self.precio = precio

    def getIdentificador(self):
        return self.identificador

    def setIdentificador(self, identificador):
        self.identificador = identificador

    def getFrecuencia(self):
        return self.frecuencia

    def setFrecuencia(self, frecuencia):
        self.frecuencia = frecuencia
